.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.37.3
+    hooks:
+      - id: pyupgrade
+        # Specify the oldest Python version that "cashews" supports.
+        args: [--py37-plus]
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-added-large-files
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: debug-statements
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        args: [--line-length=160]
```

Comment:
With isort  we have another line length
`line_length = 119` - so may be we can have the same value for black ?

Reply:
Sorry, this is a typo. I was going to write `119`, but I forgot to change the value. I will fix.